src/sedcli_util.c

```diff
@@ -111,6 +112,7 @@ void *alloc_locked_buffer(size_t size)
 	if (!buf)
 		return NULL;
 
+	memset(buf, 0, 1);	/* silence GCC warning (error) */
```

Comment:
Any reason not to zero all of buf?

Reply:
Not really, other than just to limit the amount of work done. I wanted to just do "*buf = 0;" but 'buf' is 'void *' and the casting necessary was not pretty. This path is not performance critical, so could just zero all of it. I was annoyed that I had to touch *buf at all, but trying to fix this in GCC seems excessive. It will be problem for our target distribution, though, as GCC 11 is already in use there.

Reply:
Perhaps, rather than use memset, the line should be:
```
*(char *)buf = 0;	/* silence GCC warning (error) */
```
It also turns out that the #include <string.h> is not necessary, perhaps due to side-effects of other #includes.